src/middleware/thunk.js

```diff
@@ -1,5 +1,5 @@
 export default function thunkMiddleware({ dispatch, getState }) {
-  return (next) => (action) =>
+  return next => action =>
     typeof action === 'function' ?
       action(dispatch, getState) :
```

Comment:
I thought about that, too, but 9 times out of 10 a thunk just needs `dispatch()`, and single argument form is easier.